src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -10,9 +17,27 @@
     @JsonProperty("company_number")
     private final String companyNumber;
 
-    public ChipsRequest(String objectionId, String companyNumber) {
+    @JsonProperty("attachments")
+    private final Map<String, String> attachments;
+
+    @JsonProperty("reference_number")
+    private final String referenceNumber;
+
+    @JsonProperty("customer_email")
+    private final String customerEmail;
+
+    @JsonProperty("reason")
+    private final String reason;
+
+    public ChipsRequest(String objectionId, String companyNumber,
+                        List<Attachment> attachments,
+                        String customerEmail, String reason) {
         this.objectionId = objectionId;
         this.companyNumber = companyNumber;
+        this.attachments = buildAttachmentsMap(attachments);
+        this.referenceNumber = ""; // TODO OBJ-162 add ref number
```

Comment:
At the moment, the reference number is the objection id, so the reference number should be set to the objection id value to be passed to chips.

Reply:
Agree. You could leave the TODO comment though to say change to ref number when available.